bonsai/examples/governance/Cargo.toml

```diff
@@ -1,12 +1,19 @@
 [workspace]
-default-members = [
-    "relay"
-]
 members = [
     "methods",
     "relay"
 ]
 
+[workspace.dependencies]
+# Outside of the risc0 monorepo, use the following dependency definitions.
+# risc0-build = { git = "https://github.com/risc0/risc0", branch = "release-0.16" }
+# risc0-zkvm = { git = "https://github.com/risc0/risc0", branch = "release-0.16", default-features = false }
+# bonsai-sdk = { git = "https://github.com/risc0/risc0", branch = "release-0.16" }
```

Comment:
It's legal to put both `path = ` and `version =`. This allows in-tree and out-tree building to work.